crates/astria-conductor/src/sequencer/mod.rs

```diff
@@ -285,22 +289,58 @@ async fn subscribe_new_blocks(
     .map_err(|_| ResubscriptionError::BackoffFailed)
 }
 
-fn forward_pending_block(
+fn forward_pending_block_or_resync(
+    expected_height: u32,
```

Comment:
can you add a comment clarifying `expected_height`?

Reply:
Expanded the doc comment on the function and what it does.